src/components/AccessibilityGuide/ProgressiveEnhancement.mdx

```diff
@@ -0,0 +1,109 @@
+fetched, and how data is rendered. Many issues, however, point back to DS
+components.
+
+## HTML Button Issue
+
+HTML `<button>`s can work without javascript in the following contexts.
+
+- When used inside a `<form>` element, if it has an attribute of type=”submit”,
+  the button will be clickable.
+- When used inside a `<form>` element, if it has an attribute of type=”reset”,
+  the button will be clickable and clear out input elements.
+
+There are DS components that use the `Button` component, and therefore the
+`<button>` HTML element, and cannot fully function without Javascript. These
+components rely on the button element to open, close, or perform user actions
+that won’t work without javascript.
+
+## Component Audit
+
+As of v3.6.1, the DS contains 25 fully functional components, and 36 components
+that need Javascript to work.
+
+The following (25) components are fully functional without Javascript:
+
+- `Breadcrumbs`, `Card`, `FeaturedContent`, `FieldSet`, `Footer`, `Form`,
+  `FormRow`, `FormField`, `HelperErrorText`, `Hero`, `HorizontalRule`, `Icon`,
+  `Image`, `Label`, `Link`, `List`, `Logo`, `SimpleGrid`, `SkipNavigation`,
+  `StatusBadge`, `StructuredContent`, `StyledList`, `Table`, `Template`, `Text`
+
+The following (16) use the HTML `<button>` element and are not fully functional without
+Javascript:
+
+- `Accordion`, `AlphabetFilter`, `Banner`, `Button`, `ButtonGroup`, `Header`,
+  `Menu`, `Modal`, `MultiSelect`, `MultiSelectGroup`, `NewsletterSignup`,
+  `Notification`, `Pagination`, `SubNav`, `Tabs`, `TagSet`
+
+The following (15) components need a pattern update to work without Javascript:
+
+- `CheckBox`, `CheckboxGroup`, `FeedbackBox`, `FilterBarInline`, `FilterBarPopup`,
+  `ProgressIndicator`, `Radio`, `RadioGroup`, `SearchBar`, `Select`,
+  `SkeletonLoader`, `Slider`, `TextInput`, `Toggle`, `Tooltip`
+
+The following (2) can't be fixed due to browser limitations:
+
+- `AudioPlayer`, `VideoPlayer`
+
+The following (1) depend on external libraries:
+
+- `DatePicker`
+
+## General Recommendations
+
+In our current web world and based on the number of users that have JS disabled,
+instead of fixing all no-JS issues, we should attempt to fix what we can. This
+audit gives us a better understanding of what components do not work without JS.
+We need to document our findings and be transparent about the components that do
+not work. Based on the Engineering General Javascript standard, a no-JS fallback
+is preferable but not required.
+
+Various new pages and features that are being built have JS-heavy requirements.
+Based on the standards, we need to recommend and encourage teams to add a
+noscript fallback message that instructs patrons to enable JS to use the site.
+
+We recommend rendering a simple component with the patron message to use as the
+fallback noscript solution. This would not be a React component but rather an
+HTML/CSS snippet.
```

Comment:
This is written like a TAD. I think the wording should be updated to be more direct. 

"In situations that do X, it is recommended to do Y."